### ISSUE TITLE
[BUG] Age in sample_metadata is inconsistently typed

### ISSUE THREAD

**Issue by jashapiro** [AlexsLemonade/scpca-nf#735]
**Describe the bug**

In the `metadata(sce)$sample_metadata` table, we sometimes have the `age` column as a `double` and sometimes a `character`. 

**To reproduce**
Using the processed output files for two libraries: 

```
> type(metadata(SCPCL000822)$sample_metadata$age)
[1] "double"
> type(metadata(SCPCL000608)$sample_metadata$age)
[1] "character"
```

**Expected behavior**

Always one type!

**Version information**

I believe these outputs are from the latest scpca-nf output, but I am not sure... we should check with 0.8.0 outputs

**Additional context**

The error could easily be in `scpcaTools`; I have not yet gone to look at where the data is read in.

**Comment by allyhawkins** [comment]
For some reason I thought we had fixed this, but it gets read in here and we don't specify everything be a character, which we probably should. 
https://github.com/AlexsLemonade/scpca-nf/blob/8bef82d853d19e5aeddd75401aa54cf8bfbced13/bin/generate_unfiltered_sce.R#L164

**Comment by jashapiro** [comment]
We probably fixed it somewhere else...

**Comment by allyhawkins** [comment]
Closed by #738
